components/Toggle/Toggle.jsx

```diff
@@ -40,10 +49,18 @@ const Switch = styled.div`
   ${CloseIcon},
   ${CheckIcon} {
     align-items: center;
-    background-color: ${Colors.BLACK[100]};
+    background-color: ${({
+      theme: {
+        colors: { neutral },
+      },
+    }) => `${neutral[100]};`};
     border-radius: 50%;
     display: flex;
-    color: ${Colors.BLACK[400]};
+    color: ${({
+      theme: {
+        colors: { neutral },
+      },
+    }) => `${neutral[500]};`};
```

Comment:
WDYT of putting all style rules of this component, that depend on theme props in just one callback?